test-data/unit/check-flags.test

```diff
@@ -515,3 +515,59 @@ x, y = 1, 2  # type: Unchecked, Unchecked
 [out]
 main:4: error: Type of variable becomes "Any" due to an unfollowed import
 main:6: error: A type on this line becomes "Any" due to an unfollowed import
+
```

Comment:
It'd be nice to have some tests for the basic functionality.  These tests do a good job of exercising the more interesting edge cases, but it's worth testing putting an `Any` type in a variety of expressions to make sure it's caught/the behavior is what we expect.